### ISSUE TITLE
Built for Windows?

### ISSUE THREAD

**Issue by trickkiste** [mau0x/html5-videoEditor#3]
Hi Matthieu,

like your work. Got it to work in a Docker container ... mostly. Still got a few questions though.

Seems like it was built for a Windows Environment, am I right there?
# from modules/avisynth.js

```
//LoadingPlugins
avs.push('LoadPlugin("C:\\Program Files (x86)\\AviSynth 2.5\\plugins\\ffms2.dll")');
```

Are there any other hardcoded references I need to be aware of?

PS: I am not writing all these issues to annoy you, but rather in order to post some documentation here about what I am up to, so others can profit from that information.

As soon as I have this running well in Docker, I will commit an updated Readme, so everyone can play around with your app like that.

**Comment by mau0x** [comment]
You are right, at the time I started the project the only frameserver I knew that was capable of the things i wanted to realize was AviSynth (http://avisynth.nl/index.php/Main_Page) which could only run on Windows. Apart from that, I do not remember any other dependency than ffmpeg, which should be available across platforms. As you know I do not maintain this project anymore (lack of time) so there might be a better frameserving software or approach for compositing the source files on the server side. Maybe there is something like a node.js support for after effects as there is for photoshop?
